mlflow/server/js/src/components/MetricsPlotControls.js

```diff
@@ -39,9 +39,11 @@ export class MetricsPlotControls extends React.Component {
 
   render() {
     const { chartType } = this.props;
+    const lineSmoothnessTooltipText =
+      'Sets the amount of smoothing. ' +
```

Comment:
cc @stbof do you have thoughts on the wording here? This is the tooltip that shows up next to the "Line Smoothing" option in the metrics plot

Reply:
My comments in the Jira are: It's not clear exactly what you would use the Line Smoothness control for, why the range is 0-1.30, and what the numbers mean. 

So the tooltip needs to address those comments. 

Honestly, I wonder if anyone would actually use this control.  What benefits are they getting by smoothing the line?